text/0000-extern-types.md

```diff
@@ -64,7 +64,7 @@ These types are FFI-safe. They are also DSTs, meaning that they do not implement
 In Rust, pointers to DSTs carry metadata about the object being pointed to.
 For strings and slices this is the length of the buffer, for trait objects this is the object's vtable.
 For extern types the metadata is simply `()`.
-This means that a pointer to an extern type is identical to a raw pointer.
+This means that a pointer to an extern type is identical to a raw pointer (ie. it is not a "fat pointer").
```

Comment:
Probably better to just say it is not a fat pointer and not even bother bringing up raw pointers because raw pointers can be fat too! `*const [T]` is fat for example despite being a raw pointer.